README.md

```diff
@@ -49,16 +49,15 @@ void main()
     import dparse.lexer;
     import dparse.parser : parseModule;
     import dparse.rollback_allocator : RollbackAllocator;
-    import std.string : representation;
 
     auto sourceCode = q{
         void foo() @safe {
             void bar();
         }
-    }.dup;
+    };
     LexerConfig config;
     auto cache = StringCache(StringCache.defaultBucketCount);
-    auto tokens = getTokensForParser(sourceCode.representation, config, &cache);
+    auto tokens = getTokensForParser(sourceCode, config, &cache);
 
     RollbackAllocator rba;
     auto m = parseModule(tokens, "test.d", &rba);
```

Comment:
You would need to update the URL too, but I can do this in a quick follow-up too.
OTOH we would then need to release libdparse 0.8 because run.dlang.io only downloads the latest, stable version.